app/views/licences/view.njk

```diff
@@ -39,6 +40,19 @@
     }) }}
   {% endif %}
 
+  {% if notification %}
+    <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+      <div class="govuk-notification-banner__header">
+        <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+          Important
+        </h2>
+      </div>
+      <div class="govuk-notification-banner__content">
+     <strong> {{ notification }} </strong>
+      </div>
+    </div>
```

Comment:
Unless we _really_ cannot make the Nunjucks version work, we always favour it. Our reasoning is when updates are made to the GOVUK package there is less need for us to revisit our views if we used the Nunjucks version.

So, could we try switching this to

```suggestion
    {% if notification %}
      {{ govukNotificationBanner({
        html: '<strong>' + notification + '</strong>'
      }) }}
    {% endif %}
```

> Testing locally this gave the same result.

Reply:
Mush better. Thanks.